docs/04. Guards.md

```diff
@@ -80,7 +80,7 @@ return [
 ];
 ```
 
-Because of the way Zend Framework 2 handles config, you can without problem define some rules in one module, and
+Because of the way Laminas Framework 2 handles config, you can without problem define some rules in one module, and
```

Comment:
"Laminas Framework 2" is another phrase that might be worth searching and replacing -- perhaps with simply "Laminas."